modules/javafx.graphics/src/main/java/javafx/scene/robot/package.html

```diff
@@ -1,3 +1,32 @@
+<!doctype html>
+
+<!--
+/*
+ * Copyright (c) 2012, 2017, Oracle and/or its affiliates. All rights reserved.
```

Comment:
Should be: `Copyright (c) 2018, Oracle` ...

Reply:
Fixed.